### DISCUSSION TITLE
Public portfolio with a private account

### DISCUSSION THREAD

**Discussion by Borks** [pixelfed/pixelfed#4960] [Q&A]
Is it possible to have a private account, but still enable a public portfolio. 
It seems that when I mark my account as private, it won't allow me to enable the portfolio. 
But when I keep the account open, I can't mark my old posts as private..

Is is possible to have a private-by default, but allow portfolio for public sharing somehow?
Thanks.

**Comment by smurfn33dl3** [comment]
It sounds like you're navigating the privacy settings of a platform where you can share your work or posts, and you're looking to find a balance between keeping your account private while still being able to publicly share a portfolio. The ability to do this can vary greatly depending on the specific platform you're using, as each has its own set of features and limitations regarding privacy and sharing settings.

This might help you
    Separate Portfolio Page: If the platform allows, create a separate page or section specifically for your portfolio. This page could be set to public while your main account remains private.

    External Portfolio Platform: Use an external site to host your portfolio. Platforms like GitHub (for code), Behance (for design), or a personal website can offer more flexibility in privacy and sharing settings. You can then link to this external portfolio from your private account if the platform allows linking to external sites.

    Selective Sharing: Some platforms offer a feature where you can select which posts or pieces of work are public and which are private. If available, you could manually set your portfolio items to be public while keeping other posts private.

    Public Profile with Privacy Controls: Instead of setting your entire account to private, explore if the platform has more granular privacy settings that allow you to hide specific posts or sections of your profile. This way, you can keep your account public for the portfolio's sake but still control the visibility of other content.

    Customization and Permissions: Check if the platform has options for customizing viewer permissions. For instance, you might be able to create a public portfolio that’s visible to everyone but restrict access to other parts of your account to only approved followers or connections.

    Contact Support or Community Forums: If the platform’s settings don’t offer a clear way to accomplish this, reaching out to the platform’s support team or consulting community forums might reveal workarounds or best practices other users have found.
